### PR TITLE
Bugfix: Homepage-AppBarMobile position fixed

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Describe your changes

- Adjust AppNavMobile fixed to top of the screen

## Issue ticket number and link
https://trello.com/c/2GqJquaE

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update: 
- [ ] Create storybook for components involved

**Top AppBar: Mobile-version**

Screenshot 2023-06-09 at 14 04 29

### CODE REVIEW

File: src/components/EventCard/EventCard.tsx [hunk 1]
```diff
@@ -1,167 +1,182 @@
...
-    }
-
     return dateObject.format('D MMM YYYY')
   }
 
-  //     const url = event.images && event.images.length > 0
-  //       ? event.images[0].url
-  //       : '/images/NoImage.png';
+  const url =
+    event.images && event.images.length > 0 ? event.images[0].url : eventImage
```
Comment (jlora-mck) [inline src/components/EventCard/EventCard.tsx:131]: Please refactor this so we dont repeat `event` (remember DRY principles)

File: src/components/EventList/EventList.tsx [hunk 2]
```diff
@@ -19,22 +19,22 @@ export type EventListProps = {
...
   events,
-  // onOpen,
-  // onPause,
-  // onClose,
-  onSelected,
-}: EventListProps): JSX.Element {
+}: // onOpen,
+// onPause,
+// onClose,
+//onSelected,
```
Comment (jlora-mck) [inline src/components/EventList/EventList.tsx:30]: Please remove this comments

File: src/components/EventList/EventList.tsx [hunk 3]
```diff
@@ -19,22 +19,22 @@ export type EventListProps = {
...
+EventListProps): JSX.Element {
   const classes = useStyles()
 
   // const handleOpen = () => onOpen
   // const handlePause = () => onPause
   // const handleClose = () => onClose
-  const handleSelected = (event: Conference) => onSelected(event)
+  //const handleSelected = (event: Conference) => onSelected(event)
```
Comment (jlora-mck) [inline src/components/EventList/EventList.tsx:37]: Please remove comments

File: src/components/EventList/EventList.tsx [hunk 4]
```diff
@@ -53,7 +53,7 @@ export default function EventList({
           // onOpen={handleOpen}
           // onPause={handlePause}
           // onClose={handleClose}
-          onSelectedEvent={handleSelected}
+          //onSelectedEvent={handleSelected}
```
Comment (jlora-mck) [inline src/components/EventList/EventList.tsx:56]: Please remove comments

File: src/pages/Events/Events.tsx
Comment (jlora-mck) [inline src/pages/Events/Events.tsx:1]: Why did we remove all this code ?

File: src/components/EventList/EventList.tsx (hunk 4 above)
Comment (SilviaTrujillano23) [reply src/components/EventList/EventList.tsx]: Done

File: src/components/EventList/EventList.tsx (hunk 3 above)
Comment (SilviaTrujillano23) [reply src/components/EventList/EventList.tsx]: Done

File: src/components/EventList/EventList.tsx (hunk 2 above)
Comment (SilviaTrujillano23) [reply src/components/EventList/EventList.tsx]: Done

File: src/pages/Events/Events.tsx
Comment (SilviaTrujillano23) [reply src/pages/Events/Events.tsx:1]: Code is not necessary for this Bugfix nor last User Story. Besides corresponded to refactored code already implemented (code legacy).

File: src/components/EventCard/EventCard.tsx (hunk 1 above)
Comment (SilviaTrujillano23) [reply src/components/EventCard/EventCard.tsx]: Thanks, now it is refactored and majestic :)

**Review by jlora-mck — APPROVED** [review]

**Review by kellyCSalazar — APPROVED** [review]

**Pull request merged**